src/command_modules/azure-cli-interactive/azclishell/app.py

```diff
@@ -455,9 +455,16 @@ def _special_cases(self, text, cmd, outside):
             if text[0] == SELECT_SYMBOL['outside']:
                 cmd = text[1:]
                 outside = True
-                if cmd.strip() and cmd.split()[0] == 'cd':
-                    self.handle_cd(parse_quotes(cmd))
-                    continue_flag = True
+                if cmd.strip():
+                    parsed_cmd = parse_quotes(cmd)
+                    if cmd.split()[0] == 'cd':
+                        self.handle_cd(parsed_cmd)
+                        continue_flag = True
+                    elif cmd.split()[0] == 'export' and len(parsed_cmd) > 2:  # sets environment variables
+                        env_name = parsed_cmd[1]
+                        env_val = parsed_cmd[2]
+                        os.environ[env_name] = env_val
+                        continue_flag = True
```

Comment:
No. The export syntax is more complex then this:
https://www.cyberciti.biz/faq/linux-unix-shell-export-command/

the `env_val` may contain another variable as well. For example,
```
export PATH=\bin\local\someapp:$PATH
```
Without expanding it, it won't work.